packages/cli/README.md

```diff
@@ -41,7 +41,7 @@ To develop Vercel CLI, first check out the source code, install dependencies, an
 ```bash
 git clone https://github.com/vercel/vercel.git
 cd vercel
-pnpm
+pnpm i
```

Comment:
```suggestion
pnpm install
```

Might want to use the full word here for new contributors to understand

Reply:
Fixed!